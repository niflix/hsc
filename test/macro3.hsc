<$macro foo /CLOSE bar:string="bar_foo" foo:string="foo_foo" fox:string="fox_foo">
  bar(foo)=<(bar)> foo(foo)=<(foo)> fox(foo)=<(fox)>
  <$content>
  bar(foo)=<(bar)> foo(foo)=<(foo)> fox(foo)=<(fox)>
</$macro>

<$define bar:string="bar_global">
<$define fox:string="fox_global">

<html>
  <head>
    <title>macro3</title>
  </head>
  <body>
    <h1>foo 1</h1>
    <foo></foo>
    <h1>foo 2</h1>
    <foo><$define foo:string="foo_content"></foo>
  </body>
</html>
