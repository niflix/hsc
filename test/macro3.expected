<html>
  <head>
    <title>macro3</title>
  </head>
  <body>
    <h1>foo 1</h1>
      bar(foo)=bar_foo foo(foo)=foo_foo fox(foo)=fox_foo
    bar(foo)=bar_foo foo(foo)=foo_foo fox(foo)=fox_foo
    <h1>foo 2</h1>
      bar(foo)=bar_foo foo(foo)=foo_foo fox(foo)=fox_foo
    bar(foo)=bar_foo foo(foo)=foo_foo fox(foo)=fox_foo
  </body>
</html>
